models/twilio_message_enhanced.sql

```diff
@@ -99,13 +99,13 @@ final as (
         union_messages.price,
         union_messages.price_unit,
         union_messages.updated_at,
-        addresses.iso_country
+        addresses.iso_country 
```

Comment:
This seems to be null for all records. Should we instead try and do the parsing to get the country code from the phone number? Were you able to attempt this approach?

Reply:
Yeah this is a bit tricky. I would prefer we don't use seed data for this as you mentioned this could go stale. Let's brainstorm more about this and circle back.